metabase/metabase
# Add read-only Notifications tab

Adds a new `Notifications` tab in the Account settings. It lists dashboard subscriptions and alerts you subscribed to or created by you.

How to test:
- Create a dashboard subscription or an alert
- Go to `Notifications` tab
- You should be able to see them and navigate to questions/dashboards

<img width="593" alt="Screen Shot 2021-08-18 at 6 56 58 pm" src="https://user-images.githubusercontent.com/8542534/129931362-9cbac036-2060-4ee8-b3b2-f59c91fe7a00.png">
<img width="661" alt="Screen Shot 2021-08-18 at 6 57 09 pm" src="https://user-images.githubusercontent.com/8542534/129931368-9437d42e-a888-47a4-9003-bd48cf484907.png">
